src/components/screen-header/HeaderButton.tsx

```diff
@@ -39,21 +39,23 @@ const HeaderButton: React.FC<HeaderButtonProps> = (buttonProps) => {
 };
 
 export type HeaderButtonWithoutNavigationProps = {
-  text: string;
+  text?: string;
+  icon?: React.ReactNode;
```

Comment:
Sometime ago it was decided that we always use text in headers in stead of icons. Talked with Christian, and it was decided we still follow this rule also for the departures bottom sheet. He will update the sketches to use "Close"/"Lukk" instead.